app/views/reports/regions/_search_script.html.erb

```diff
@@ -0,0 +1,16 @@
+<script>
+  const search = new RegionsSearch().search;
+  const form = $("#search-form")
+  form.on("submit", function(event) {
+    input = $(event.target).find(".typeahead-input")
+    search(input);
+    return false;
+  })
+  form.on("keyup", function(event) {
+    search(event.target);
+    return false;
+  })
+
+  const async = false
+  new AdminAccess("facility-access").initialize(async)
```

Comment:
@claudiovallejo You'll want to add a guard here to only create `AdminAccess` if `facility-access` is on the page.  Otherwise this will break.

Reply:
We can move these last two lines back to the index page, since the access stuff is specific to that page.